Intersect.Editor/Localization/Strings.cs

```diff
@@ -642,6 +642,8 @@ public partial struct AnimationEditor
 
             public static LocalizedString simulatedarkness = @"Simulate Darkness: {00}";
 
+            public static LocalizedString camerashake = @"Camera Shake (ms): {00}";
```

Comment:
Don't forget to add: 

`[JsonProperty(NullValueHandling = NullValueHandling.Ignore)]`

Also use Pascal casing (C# Coding Conventions)